pkg/provider/aws/action/kind/kind.go

```diff
@@ -46,23 +46,28 @@ type kindRequest struct {
 	allocationData *allocation.AllocationData
 }
 
+type KindResultsMetadata struct {
+	Username   string  `json:"username"`
+	PrivateKey string  `json:"private_key"`
+	Host       string  `json:"host"`
+	Kubeconfig string  `json:"kubeconfig"`
+	SpotPrice  float64 `json:"spot_price,omitempty"`
+}
+
 // Create orchestrate 3 stacks:
 // If spot is enable it will run best spot option to get the best option to spin the machine
 // Then it will run the stack for windows dedicated host
-func Create(ctx *maptContext.ContextArgs, args *KindArgs) error {
-	// Create mapt Context
+func Create(ctx *maptContext.ContextArgs, args *KindArgs) (*KindResultsMetadata, error) {
 	if err := maptContext.Init(ctx, aws.Provider()); err != nil {
-		return err
+		return &KindResultsMetadata{}, err
```

Comment:
As returned values are *KindResultsMetadata, error...this should return nil, err

same for other returns with err